deploy/sre-build-test/50-build-test.CronJob.yaml

```diff
@@ -79,6 +85,12 @@ spec:
                     ;;
                   Complete)
                     echo "$(date): Build Complete"
+                     oc -n "${JOB_NS}" get job --no-headers | \
```

Comment:
if this fails, the build fails. I this the right behaviour?

Reply:
not sure, but will add an `||true to it`

Reply:
done